app/views/docs/functions.phtml

```diff
@@ -40,186 +40,149 @@ $runtimes = $this->getParam('runtimes', []);
+
+  const trigger = req.env.APPWRITE_FUNCTION_TRIGGER;
+
+  res.json({
+    message: 'Hello from Appwrite!',
+    payload,
+    secretKey,
+    randomNumber,
+    trigger,
+  });
+};</code></pre>
         </div>
         <b>Installing Dependencies</b>
-        <p>Include a <b>package.json</b> along with your function, and Appwrite handles the rest! The best practice is to make sure <b>node_modules</b> folder is <b>not</b> part of your tarball. This example uses the <a href="https://www.npmjs.com/package/node-fetch">node-fetch</a> package</p>
+        <p>Include a <b>package.json</b> along with your function, and Appwrite handles the rest! The best practice is to make sure <b>node_modules</b> folder is <b>not</b> part of your tarball.</p>
         </div>
     </li>
     <li>
         <h3>PHP</h3>
         <div>
         <div class="ide margin-top-small" data-lang="php" data-lang-label="PHP">
-            <pre class="line-numbers"><code class="prism language-php" data-prism>&lt;?php 
-
-return function($request, $response) {
-    $payload = json_decode($request['payload']);
-    $env = $request['env'];
-
-    $randomNumber = rand(1, 100);
-    $todo = json_decode(file_get_contents("https://jsonplaceholder.typicode.com/todos/$randomNumber"), true);
-
-    $task = array_merge($todo, [
-        'message' => 'Hello from Appwrite Functions 👋',
-    ]);
-
-    switch($env['APPWRITE_FUNCTION_TRIGGER']) {
-        case 'http': 
-            $task['trigger'] = 'HTTP'; 
-            break;
-        case 'schedule': 
-            $task['trigger'] = 'CRON'; 
-            break;
-        case 'event': 
-            $task['trigger'] = 'Event'; 
-            break;
-        default: 
-            $task['trigger'] = 'Unknown'; 
-            break;
-    }
+            <pre class="line-numbers"><code class="prism language-php" data-prism>&lt;?php
+
+return function ($req, $res) {
+  $payload =
+    $req['payload'] ?:
+    'No payload provided. Add custom data when executing function.';
+
+  $secretKey =
+    $req['env']['SECRET_KEY'] ?:
+    'SECRET_KEY environment variable not found. You can set it in Function settings.';
 
-    $response->json($task);
+  $randomNumber = \mt_rand() / \mt_getrandmax();
+
+  $trigger = $req['env']['APPWRITE_FUNCTION_TRIGGER'];
+
+  $res->json([
+    'message' => 'Hello from Appwrite!',
+    'payload' => $payload,
+    'secretKey' => $secretKey,
+    'randomNumber' => $randomNumber,
+    'trigger' => $trigger,
+  ]);
 };</code></pre>
         </div>
         <b>Installing Dependencies</b>
-        <p>Adding dependencies to PHP requires you to have a <b>composer.json</b> file. The best practice is to make sure <b>vendor</b> directory is <b>not</b> part of your tarball. </p>
+        <p>Include a <b>composer.json</b> file in your function, make sure to require autoload.php from vendor folder, and Appwrite will handle the rest!. The best practice is to make sure <b>vendor</b> directory is <b>not</b> part of your tarball. </p>
```

Comment:
Just bringing up that Eldad didn't enjoy calling out deployment package a tarball. Did we ever come up with another name?